.kokoro/release/publish_javadoc11_maven.cfg

```diff
@@ -0,0 +1,31 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# cloud-rad dev
+# TODO: Update this or publish_javadoc11.sh to the prod bucket
+env_vars: {
+  key: "STAGING_BUCKET_V2"
+  value: "docs-staging-v2-dev"
```

Comment:
FYI, when we run this in parallel, then maven version will upload to the dev bucket. I can stage this to the dev devsite and see how it looks.